ant-networking/src/lib.rs

```diff
@@ -775,7 +775,7 @@ impl Network {
     pub async fn get_local_quoting_metrics(
         &self,
         key: RecordKey,
-    ) -> Result<(QuotingMetrics, bool)> {
+    ) -> Result<Option<(QuotingMetrics, bool)>> {
```

Comment:
What does None mean in this case? I think `Result<Option<_>>` is a confusing pattern